File: code/sorting/unclassified/src/average/average.js
/* Part of Cosmos by OpenGenus Foundation */

// finds the average of an array of numbers.
//Using map function
function getAverage(numbers) {
  var sum = 0;
  numbers.map(function(number) {
    sum += number;
  });
  return sum / numbers.length;
}

nums = [10, 20, 30, 40, 50];
console.log(getAverage(nums));

File: code/sorting/unclassified/src/average/average.py
## Part of Cosmos by OpenGenus Foundation
## Find of average of numbers in an array
## Contributed by: Pranav Gupta (foobar98)

n = int(input("Enter no. of elements: "))
a = []  # empty list

for i in range(0, n):
    # every number in a new line
    x = int(input("Enter number: "))
    a.append(x)

avg = sum(a) / n
print("Average of numbers in list: " + str(round(avg, 2)))
